business/definitionService.js

```diff
@@ -123,13 +126,13 @@ class DefinitionService {
   async compute(coordinates, curationSpec) {
     const curation = await this.curationService.get(coordinates, curationSpec)
     const raw = await this.harvestService.getAll(coordinates)
-    const facets = await this._computeFacets(coordinates, raw, curation)
-    const summarized = await this.summaryService.summarizeAll(coordinates, raw, facets || {})
+    const summarized = await this.summaryService.summarizeAll(coordinates, raw)
     const aggregated = await this.aggregationService.process(coordinates, summarized)
     const definition = await this.curationService.apply(coordinates, curation, aggregated)
+    this._ensureFacets(coordinates, definition)
```

Comment:
`_ensureFacets` only accepts `definition` as a parameter

Reply:
D'oh figured it out. the tests are all calling `_ensureFacets` directly. That was a bit of a hack but I didn't want to setup and run through the whole service. lazy. I will update the tests to use more mocks and run the whole service.